server/boot/routes.js

```diff
@@ -29,18 +30,29 @@ module.exports = function(app) {
       password: req.body.password
     }, 'user', function(err, token) {
       if (err) {
-        res.render('response', {
-          title: 'Login failed',
-          content: err,
-          redirectTo: '/',
-          redirectToLinkText: 'Try again'
-        });
+        if(err.details && err.code === 'LOGIN_FAILED_EMAIL_NOT_VERIFIED'){
+          res.render('reponseToTriggerEmail', {
+            title: 'Login failed',
+            content: err,
+            redirectToEmail: '/api/users/'+ err.details.userId + '/verify',
+            redirectTo: '/',
+            redirectToLinkText: 'Click here',
+            userId: err.details.userId
+          });
+        } else{
+          res.render('response', {
+            title: 'Login failed. Wrong username or password',
+            content: err,
+            redirectTo: '/',
+            redirectToLinkText: 'Please login again',
+          });
+        }
         return;
       }
-
       res.render('home', {
         email: req.body.email,
-        accessToken: token.id
+        accessToken: token.id,
+        redirectUrl: 'api/users/change-password?access_token=' + token.id
```

Comment:
While this seems to work correctly, I think it would be safer to use an absolute path here, i.e. `/api/users/change-password?access_token=`.